pages/admin.js

```diff
@@ -36,12 +45,22 @@ const Admin = ({ error, wards }) => {
 export const getServerSideProps = propsWithContainer(
   verifyAdminToken(
     async ({ container, authenticationToken }) => {
-      const { wards, error } = await container.getRetrieveWards()(
+      const wardsResponse = await container.getRetrieveWards()(
         authenticationToken.trustId
       );
+      const trustResponse = await container.getRetrieveTrustById()(
+        authenticationToken.trustId
+      );
+
+      const trustName = trustResponse.trust ? trustResponse.trust.name : null;
 
       return {
-        props: { wards: wards, error: error },
+        props: {
+          wards: wardsResponse.wards,
+          trust: { name: trustName },
```

Comment:
I may be wrong, but I think this could be inlined as
```javascript
{ name: trustResponse.trust?.name }
```

I can't remember if the project is set up to use the newest syntax or not.